jfrog-applications/frogbot/get-started/README.md

```diff
@@ -1,19 +1,18 @@
 # Get Started
 
-JFrog Frogbot is a Git bot that scans your git repositories for security vulnerabilities.
+JFrog Frogbot is a Git bot that scans your git repositories for security vulnerabilities through the following key functions:
 
-1. It scans pull requests immediately after they are opened but before they are merged. This process notifies you if the pull request is about to introduce new vulnerabilities to your code. This unique capability ensures that the code is scanned and can be fixed even before vulnerabilities are introduced into the codebase.
-2. It scans the Git repository periodically and creates pull requests with fixes for vulnerabilities that are detected.
+1. Scans repository selected branches periodically and creates pull requests with fixes for vulnerabilities that are detected.
+2. Scans pull requests making sure no new vulnerabilities are about to enter to your code. 
```

Comment:
Scans pull requests making sure no new vulnerabilities are about to enter to your code. 
-->
Scans mewly opened pull requests to ensure no new vulnerabilities are introduced to the codebase.